clvm-utils/src/curry_tree_hash.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{tree_hash_atom, tree_hash_pair};
+
+pub fn curry_tree_hash(program_hash: [u8; 32], arg_hashes: &[[u8; 32]]) -> [u8; 32] {
+    let nil = tree_hash_atom(&[]);
+    let one = tree_hash_atom(&[1]);
+    let op_q = one;
```

Comment:
```suggestion
    let op_q = tree_hash_atom(&[1]);
```
it looks a little bit redundant